whatshap/variants.py

```diff
@@ -261,6 +261,16 @@ def _alignments_to_reads(self, alignments, variants, sample, reference):
                 barcode = alignment.bam_alignment.get_tag("BX")
             except KeyError:
                 barcode = ""
+                
+            try:
+                hp = alignment.bam_alignment.get_tag("HP")
+            except KeyError:
+                hp = -1
+            
+            try:
+                ps = int(alignment.bam_alignment.get_tag("PS"))
```

Comment:
It's a good question! I wrote this command assuming that I am working with the output of WhatsHap. However, I think making PS a string would be better, as it would not cause errors in parts of the code that don't rely on the assumption that this tag is an integer.

Reply:
I'd be okay with that as long as there isn't a standard we should follow. Maybe we can get some help. 

@tobiasmarschall do you know what type PS "should" be or who we might ask before we switch to string?

Reply:
Went through the hiphase code and Matt uses an int there.

Reply:
The VCF specs say `PS` is an int (see Table 2 in https://samtools.github.io/hts-specs/VCFv4.3.pdf). So I'm in favor of ensuring it is an int (and giving an appropriate error message if it's not).

Reply:
Done! Thanks for pointing me to the vcf spec @tobiasmarschall. I was checking the sam tag spec...